build/android-common.sh

```diff
@@ -206,7 +206,7 @@ get_android_arch() {
 }
 
 get_common_includes() {
-    echo "-I${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include -I${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/local/include"
+    echo "-isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include -isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/include/${TARGET_HOST} -isystem ${ANDROID_NDK_ROOT}/toolchains/llvm/prebuilt/${TOOLCHAIN}/sysroot/usr/local/include"
```

Comment:
I intended to follow closer the way CFLAGS are constructed in [platform-android.mk](https://github.com/tanersener/mobile-ffmpeg/blob/2bc8e925e4a36b3a67c72f05b33b8619ca36955f/src/openh264/build/platform-android.mk#L47). Actually, with modern NDK (r20+), these `-isystem` fine-tunings are not necessary, we can rely on **clang** to work correctly with its `--sysroot`.

Reply:
Tested this version with all external libraries and didn't get any failures. It looks like new NDK is taking care of them, as you've said.